### PR TITLE
(2767) Fix title tags

### PR DESCRIPTION
**Pull request by yndajas**
## Changes in this PR

This adds titles to the home and activities pages for all users, and also makes the titles for activity form pages match the main heading of the page (usually the question, except on pages with multiple inputs)

In doing so, it refactors and cleans up a bunch of code:
- less redundancy both in terms of repetiton (DRY) and old, now unsed translations
- alphabetical ordering of remaining translations

### Before

Screenshot 2023-01-12 at 16 29 44

### After

Screenshot 2023-01-12 at 16 29 09

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by yndajas** [comment]
[Fix <title> tags in the service](https://trello.com/c/FvN3ANmz/2767-fix-title-tags-in-the-service)

File: app/views/activity_forms/objectives.html.haml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 = render layout: "wrapper" do |f|
-  = f.govuk_fieldset legend: { text: t("form.legend.activity.objectives", level: t("page_content.activity.level.#{f.object.level}")), tag: :h1, size: "xl" } do
```
Comment (yndajas) [inline app/views/activity_forms/objectives.html.haml:2]: This is intentional. The fieldset shouldn't be necessary when there's only one input. The label just said "Aims/Objectives of the activity", which adds nothing to the question "What are the aims/objectives of this %{level}?", so the label of the remaining text area is now the full question and the original label is removed

File: config/locales/models/implementing_organisation.en.yml [hunk 2]
```diff
@@ -8,11 +8,6 @@ en:
         button: Remove
         success: Implementing organisation successfully removed
 
-  form:
```
Comment (Gweaton) [inline config/locales/models/implementing_organisation.en.yml:11]: This is the only one that's making me think twice - I wonder if we should move all of the translations in this file to the `activities` translations file if we're classifying Implementing orgs as part of the activity journey?

File: app/controllers/activity_forms_controller.rb [hunk 3]
```diff
@@ -151,12 +151,15 @@ def assign_default_collaboration_type_value_if_nil
...
   def page_title(step)
-    if step == :linked_activity
-      @activity.is_oda ? t("page_title.activity_form.show.linked_non_oda_activity") : t("page_title.activity_form.show.linked_oda_activity")
-    elsif step == :identifier && @activity.programme?
+    if step == :identifier && @activity.programme?
       t("page_title.activity_form.show.identifier_level_b")
     else
-      t("page_title.activity_form.show.#{step}", sector_category: t("activity.sector_category.#{@activity.sector_category}"), level: t("page_content.activity.level.#{@activity.level}"))
+      t(
+        "page_title.activity_form.show.#{step}",
```
Comment (Gweaton) [inline app/controllers/activity_forms_controller.rb:158]: Do you think we could add a bit more to the commit message here to say that these arguments will be ignored in most cases and only used where we interpolate the `linked_activity_type` (for example)? At a quick glance, it feels as though we actually care about their values, when we don't really.

**Review by Gweaton — APPROVED** [review]
Thanks for this, a good refactor as well, I like the move towards more consistency. Just a couple of questions/thoughts

**Pull request merged**